src/main/java/blockrenderer6343/client/world/ObserverWorld.java

```diff
@@ -177,6 +178,9 @@ private int estimateTier(IConstructable multi) {
         do {
             holo.stackSize = tier + 1;
             hasChanged = false;
+            if (multi instanceof INEIPreviewModifier modifier) {
```

Comment:
This needs a `BlockRenderer6343.isGTLoaded` check right? Or are we past the point of this mod working without GT